papers/yellow/block.tex

```diff
@@ -0,0 +1,67 @@
+\section{Ergo Block Structure}
+Ergo block consists of 4 parts:
+
+\begin{itemize}
+    \item{\em Header } - minimal amount of data, required to synchronize the chain and check PoW correctness.
+    Also contains hashes of other sections.
+    \item{\em BlockTransactions } - sequence of transactions, included in this block.
+    \item{\em ADProofs } - proofs for transactions, included into BlockTransactions.
+    Allows light clients to verify all the transactions and calculate new root hash.
+    \item{\em Extension } - additional data, that does not correspond to previous sections.
+          May contain interlinks, current miner votes and (sometimes) current parameters of the chain.
+\end{itemize}
+
+\subsection{Header}
+\vspace{1em}
+\begin{tabular}{ |p{2.5cm}||p{0.5cm}|p{7.5cm}|  }
+    \hline
+    \hline
+    Field & Size & Description  \\
+    \hline
+    version  &  1 &  block version, to be increased on every soft- and hardfork  \\
+    \hline
+    parentId &  32 &  id of parent block  \\
+    \hline
+    ADProofsRoot &  32 &  hash of ADProofs for transactions in a block \\
+    \hline
+    stateRoot &  32 &  root hash (for an AVL+ tree) of a state after block application  \\
+    \hline
+    transactionsRoot  &  32 &  root hash (for a Merkle tree) of transactions in a block  \\
+    \hline
+    timestamp &  8 &  block timestamp(in milliseconds since beginning of Unix Epoch)  \\
+    \hline
+    nBits &  8 & current difficulty in a compressed view  \\
+    \hline
+    height &  4 & block height  \\
+    \hline
+    extensionHash & 32 & root hash of extension section  \\
+    \hline
+    equihashSolution & 128 & solution of equihash PoW puzzle  \\
+    \hline
+\end{tabular}
+
+\vspace{1em}
+Some of this fields may be calculated by node by itself if it is in correct mode:
+
+\begin{itemize}
+    \item parentId: if(status==bootstrap AND PoPoWBootstrap == false).
+    \item interlinksRoot: if(PoPoWBootstrap == false).
+    \item ADProofsRoot: if(status==regular AND ADState==false AND BlocksToKeep>0).
+    \item stateRoot: if(status==regular AND ADState==false AND BlocksToKeep>0).
+\end{itemize}
+
+\subsection{Extension}
+
+Extension is a key-value storage for a variety of data.
+It contains 2 parts:
+\begin{itemize}
+    \item{\em Mandatory fields } - fields, that are known to all node in the network and may be changed
```

Comment:
fields known to all the nodes in the network 

(but you probably want to say "fields which values are set via consensus rules"